ocs_ci/deployment/baremetal.py

```diff
@@ -36,6 +36,7 @@ def __init__(self):
 
     class OCPDeployment(BaseOCPDeployment):
         def __init__(self):
+            self.flexy_deployment = True
```

Comment:
Should we instantiate this attribute in `BaseOCPDeployment` (OCPDeployment class) and set to `False`? By doing this we can change the `getattr` calls to simple attribute checks.

```python
if not getattr(self, 'flexy_deployment', False):
```
becomes
```python
if not self.flexy_deployment:
```

WDYT?

Reply:
Yep, I agree... it will be surely nicer.

Reply:
Changed in dfbb9da925d505e1b6720008f79330ebae166cd2.